src/SFA.DAS.ApplyService.InternalApi/Controllers/GatewaExperienceAndAccreditationController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.InternalApi.Infrastructure;
+
+namespace SFA.DAS.ApplyService.InternalApi.Controllers
+{
+    [Authorize]
+    public class GatewaExperienceAndAccreditationController : Controller
```

Comment:
Typo in controller name

Reply:
The controller name seems to be specific to the Gateway page it's serving, I'm not sure that's the right way as it stops them being reused.

Reply:
It could just be as simple as removing the word Gateway from the controller